tsl/test/sql/plan_skip_scan_compressed.sql.in

```diff
@@ -0,0 +1,229 @@
+-- This file and its contents are licensed under the Timescale License.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-TIMESCALE for a copy of the license.
+
+\set PREFIX 'EXPLAIN (ANALYZE, VERBOSE, COSTS OFF, SUMMARY OFF, TIMING OFF)'
+
+--DROP TABLE sensor_data;
+--DROP TABLE sensor_data_ht cascade;
+
+CREATE TABLE sensor_data(
+time timestamptz not null,
+sensor_id integer not null,
+cpu double precision null,
+temperature double precision null );
+
+INSERT INTO sensor_data (time, sensor_id, cpu, temperature) VALUES ('2000-01-01 00:00:00-00', 1, 0.5, 80);
+INSERT INTO sensor_data (time, sensor_id, cpu, temperature) VALUES ('2000-01-01 00:00:00-01', 2, 0.6, 70);
+INSERT INTO sensor_data (time, sensor_id, cpu, temperature) VALUES ('2000-02-01 00:00:00-00', 3, 0.9, 170);
+INSERT INTO sensor_data (time, sensor_id, cpu, temperature) VALUES ('2000-02-01 00:00:00-04', 4, 0.8, 140);
+INSERT INTO sensor_data (time, sensor_id, cpu, temperature) VALUES ('2000-03-01 00:00:00-00', 5, 0.9, 150);
+
+
+CREATE TABLE sensor_data_ht(
+time timestamptz not null,
+sensor_id integer not null,
+cpu double precision null,
+temperature double precision null );
+
+SELECT FROM CREATE_hypertable('sensor_data_ht','time', chunk_time_interval=> INTERVAL '30 day');
+
+INSERT INTO sensor_data_ht
+SELECT * FROM sensor_data;
+
+SELECT show_chunks('sensor_data_ht');
+
+CREATE INDEX ON sensor_data_ht (sensor_id);
+CREATE INDEX ON sensor_data (sensor_id);
+
+ALTER TABLE sensor_data_ht SET(timescaledb.compress, timescaledb.compress_orderby='time', timescaledb.compress_segmentby='sensor_id');
+
+VACUUM sensor_data;
+VACUUM sensor_data_ht;
+
+SET enable_seqscan = FALSE;
+:PREFIX SELECT DISTINCT ON (sensor_id)* FROM sensor_data_ht;
+SET enable_seqscan = TRUE;
+
+SELECT compress_chunk(i) FROM show_chunks('sensor_data_ht')i;
+
+:PREFIX SELECT DISTINCT  sensor_id FROM sensor_data_ht;
+:PREFIX SELECT DISTINCT  sensor_id FROM sensor_data_ht LIMIT 5;
+:PREFIX SELECT DISTINCT  sensor_id FROM sensor_data_ht ORDER BY 1;
+:PREFIX SELECT DISTINCT  sensor_id FROM sensor_data_ht ORDER BY 1 LIMIT 5;
+:PREFIX SELECT DISTINCT  sensor_id FROM sensor_data_ht ORDER BY 1 DESC;
+:PREFIX SELECT DISTINCT  sensor_id FROM sensor_data_ht ORDER BY 1 DESC LIMIT 5;
+
+:PREFIX SELECT DISTINCT  sensor_id FROM sensor_data_ht WHERE sensor_id > 5 ;
+:PREFIX SELECT DISTINCT  sensor_id FROM sensor_data_ht WHERE sensor_id > 5 LIMIT 5;
+:PREFIX SELECT DISTINCT  sensor_id FROM sensor_data_ht WHERE sensor_id > 5 ORDER BY 1;
+:PREFIX SELECT DISTINCT  sensor_id FROM sensor_data_ht WHERE sensor_id > 5 ORDER BY 1 LIMIT 5;
+:PREFIX SELECT DISTINCT  sensor_id FROM sensor_data_ht WHERE sensor_id > 5 ORDER BY 1 DESC;
+:PREFIX SELECT DISTINCT  sensor_id FROM sensor_data_ht WHERE sensor_id > 5 ORDER BY 1 DESC LIMIT 5;
+
+:PREFIX SELECT DISTINCT ON (sensor_id)sensor_id, time FROM sensor_data_ht;
```

Comment:
We have almost no tests that return the actual result, not only the query plan. Probably we can put these queries into a separate file and run it twice with a different `:PREFIX`, to see both the EXPLAIN and the result.

Reply:
I see that you added the queries with EXPLAIN ANALYZE, but still no plain SELECT queries without EXPLAIN, that show the actual result. Can you please add plain SELECTs as well, so that we can see the actual result in  the test output? To avoid duplicating each query by hand, put them in a separate file and include them, defining the PREFIX to be empty or to contain EXPLAIN. The same way it's done for normal skip scan with `skip_scan_query.sql`.